woocommerce-services.php

```diff
@@ -1270,8 +1283,8 @@ public function add_tracking_info_to_emails( $order, $sent_to_admin, $plain_text
 
 			// Abort if no $order was passed, if the order is not marked as 'completed' or if another extension is handling the emailing.
 			if ( ! $order
-				 || ! $order->has_status( 'completed' )
-				 || ! WC_Connect_Extension_Compatibility::should_email_tracking_details( $order->get_id() ) ) {
+				|| ! $order->has_status( 'completed' )
+				|| ! WC_Connect_Extension_Compatibility::should_email_tracking_details( $order->get_id() ) ) {
```

Comment:
We don't actually do anything here - it's just the pre-commit cleanup logic that is making changes.